Port `argmax` to structured kernels.

ysiraichi opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #60364 Port `argmin` kernel to structured kernels.
* **#60363 Port `argmax` to structured kernels.**
* #60362 Using meta checks for unary `torch.all` and `torch.any`.

Tracking issue: #55070

This PR was openned so as to solve the CI failures in main when merging: #59371 #59372 #59373 #59937 #59938.

Differential Revision: [D29265857](https://our.internmc.facebook.com/intern/diff/D29265857)

ysiraichi committed:
Port `argmax` to structured kernels.

Tracking issue: #55070

This PR was openned so as to solve the CI failures in main.

[ghstack-poisoned]

ysiraichi committed:
Update on "Port `argmax` to structured kernels."


Tracking issue: #55070

This PR was openned so as to solve the CI failures in main when merging: #59371 #59372 #59373 #59937 #59938.

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29265857).

ysiraichi committed:
Update on "Port `argmax` to structured kernels."


Tracking issue: #55070

This PR was openned so as to solve the CI failures in main when merging: #59371 #59372 #59373 #59937 #59938.

Differential Revision: [D29265857](https://our.internmc.facebook.com/intern/diff/D29265857)

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29265857).

ezyang committed:
Update on "Port `argmax` to structured kernels."


Tracking issue: #55070

This PR was openned so as to solve the CI failures in main when merging: #59371 #59372 #59373 #59937 #59938.

Differential Revision: [D29265857](https://our.internmc.facebook.com/intern/diff/D29265857)

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29265857).

ysiraichi committed:
Update on "Port `argmax` to structured kernels."


Tracking issue: #55070

This PR was openned so as to solve the CI failures in main when merging: #59371 #59372 #59373 #59937 #59938.

Differential Revision: [D29265857](https://our.internmc.facebook.com/intern/diff/D29265857)

[ghstack-poisoned]

ezyang committed:
Update on "Port `argmax` to structured kernels."


Tracking issue: #55070

This PR was openned so as to solve the CI failures in main when merging: #59371 #59372 #59373 #59937 #59938.

Differential Revision: [D29265857](https://our.internmc.facebook.com/intern/diff/D29265857)

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29265857).
